lib/adapters/REST/rest-adapter.ts

```diff
@@ -70,20 +80,9 @@ export class RestAdapter implements Adapter {
       throw new Error('Unknown endpoint')
     }
 
-    const requiredHeaders = {
-      'Content-Type': 'application/vnd.contentful.management.v1+json',
+    return await endpoint(this.axiosInstance, params, payload, {
+      ...headers,
       'X-Contentful-User-Agent': userAgent,
```

Comment:
Doesn't this always overwrite the header set in line 52?

Reply:
No it only overwrites if defined